tests/e2e/upgrade/admin-user.wdio-spec.js

```diff
@@ -6,11 +6,11 @@ describe('admin users', () => {
     const adminUser = {
       username: 'admin2',
       password: 'medic.123',
-      roles: [ 'admin' ],
+      roles: ['admin'],
       contact: { name: 'Philip' },
       place: { name: 'place', type: 'district_hospital' },
     };
```

Comment:
Please use factory to create the user.
It can be like this:

```js
const adminUser = userFactory.build({
      roles: ['admin'],
      contact: { name: 'Philip' },
      place: { name: 'place', type: 'district_hospital' },
    });
```